libexec/rtld-elf/aarch64/reloc.c

```diff
@@ -524,6 +524,9 @@ reloc_plt(Plt_Entry *plt, int flags, RtldLockState *lockstate)
 #endif
 			obj->irelative = true;
 			break;
+		case R_AARCH64_RELATIVE:
+			*where = (Elf_Addr)(obj->relocbase + rela->r_addend);
+			break;
```

Comment:
We can probably put this under the `#else` branch of the `#ifdef __CHERI_PURE_CAPABILITY__` since we should never see this under purecap.